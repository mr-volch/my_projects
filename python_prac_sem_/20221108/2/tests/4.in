ku
ku
sh
ka
112233
(8,8),(-100,-100),(0,0)
(1,2),(3,4),(0,0)
