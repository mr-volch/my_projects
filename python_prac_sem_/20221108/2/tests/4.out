Invalid input
Invalid input
Invalid input
Invalid input
Invalid input
Not a triangle
1.00
